apps/src/applab/applab.js

```diff
@@ -1351,7 +1374,7 @@ Applab.serializeToLevelHtml = function () {
   var divApplab = document.getElementById('divApplab');
   var clone = divApplab.cloneNode(true);
   // Remove unwanted classes added by jQuery.draggable.
-  $(clone).find('*').removeAttr('class');
+  $(clone).find('*').removeClass('ui-draggable ui-draggable-handle');
```

Comment:
Actually, even that still gives us trouble, since jQuery doesn't believe the cloned copy to have been turned into a draggable: "cannot call methods on draggable prior to initialization; attempted to call method 'destroy'". I'm going to stick with my original approach for now, along with a comment as to why.